owner/src/main/java/org/the_chance/honeymart/ui/addCategory/AddCategoryScreen.kt

```diff
@@ -0,0 +1,207 @@
+                    modifier = Modifier.align(Alignment.TopEnd)
+                )
+
+                Column(modifier = Modifier.fillMaxSize()) {
+                    HeaderText(title = "Add New category")
+
+                    HoneyTextField(
+                        text = state.nameCategory,
+                        modifier = Modifier.padding(top = MaterialTheme.dimens.space64),
+                        hint = "Category Name",
+                        onValueChange =  listener::changeNameCategory
+                    )
+
+                    Text(
+                        modifier = Modifier.padding(
+                            start = MaterialTheme.dimens.space16,
+                            top = MaterialTheme.dimens.space32
+                        ),
+                        text = "Select category image",
+                        style = Typography.bodyMedium.copy(color = blackOn37)
+                    )
+
+                    LazyVerticalGrid(
+                        columns = GridCells.Fixed(7),
+                        verticalArrangement = Arrangement.spacedBy(MaterialTheme.dimens.space8),
+                        horizontalArrangement = Arrangement.spacedBy(MaterialTheme.dimens.space8),
+                        modifier = Modifier.padding(horizontal = MaterialTheme.dimens.space24 )
+                    ){
+                        items(count=state.categoryImages.size ){index ->
+                            CategoryImage(
+                                iconPainter = painterResource(id = state.categoryImages[index].image),
+                                isSelected = state.categoryImages[index].isSelected,
+                                categoryImageID = state.categoryImages[index].categoryImageId,
+                                onClick = {
+                                    listener.onClickCategoryImage(
+                                        state.categoryImages[index].categoryImageId
+                                    )
+                                }
+                            )
+                        }
+                    }
+
+                }
+                HoneyFilledIconButton(
+                    label = "Add",
+                    onClick ={
+                        listener.onClickAddCategory()
+                    },
+                    isEnable = !state.isLoading,
+                    iconPainter = painterResource(id = R.drawable.icon_add_product),
+                    modifier = Modifier
+                        .align(Alignment.BottomCenter)
+                        .padding(
+                            bottom = MaterialTheme.dimens.space64,
+                            start = MaterialTheme.dimens.space24,
+                            end = MaterialTheme.dimens.space24
+                        )
+                )
+            }
+        }
+    }
+    Loading(state = state.isLoading)
+}
+
+@Preview
+@Composable
+fun HoneyMartTitle(modifier: Modifier =Modifier){
```

Comment:
Why this composable with file screen and dont use notation preview with origin composable